pkg/get/get.go

```diff
@@ -146,43 +157,38 @@ func getURLByGithubTemplate(tool Tool, os, arch, version string) (string, error)
 
 	downloadName := strings.TrimSpace(buf.String())
 
+	version, err = getToolVersion(&tool, version)
+	if err != nil {
+		return "", err
+	}
 	return getBinaryURL(tool.Owner, tool.Repo, version, downloadName), nil
 }
 
-func findGitHubRelease(owner, repo string) (string, error) {
-	url := fmt.Sprintf("https://github.com/%s/%s/releases/latest", owner, repo)
+func (tool Tool) findLastGitHubVersion() (string, error) {
+	fmt.Println("Retrieving last known version in Github for", tool.Name)
+	url := fmt.Sprintf("https://api.github.com/repos/%s/%s/releases/latest", tool.Owner, tool.Repo)
```

Comment:
I was not aware of this problem ...
I had started using the github API thinking it would be easier to write. But I was not aware that it could cause problems with quota limitations. I'm going to revert what I have started using what exists.